src/components/StyleCard/StyleCard.module.css

```diff
@@ -0,0 +1,34 @@
+.styleCard {
+  /* change DatoCMS border-color var to easily hide default section UI:before */
+  --border-color: none;
+  position: relative;
+  margin-block-end: var(--spacing-l);
+  padding: var(--spacing-l);
+  border-radius: 4px;
+  box-shadow: 0 2px 5px #0000001a;
+  transition: border 0.2s var(--material-ease);
+}
+
+.styleCard[data-status="invalid"] {
+  border: 1px solid var(--alert-color);
+}
+
+.content {
+  /* reset default DatoCMS border color */
+  --border-color: #f0f0f0;
```

Comment:
Might it be wise to map the DatoCMS border-color to another value so that you can restore it to it's actual value instead of hardcoding one?